.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   rust-stable:
     docker:
-      - image: cimg/rust:1.51.0
+      - image: cimg/rust:1.54.0
```

Comment:
1.55 is available already

Reply:
I think we can keep CircleCI 1-2 versions behind, just to provide more dissimilarity in testing and notice when the language features we are using are too new.